apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-item/index.tsx

```diff
@@ -44,14 +44,9 @@ export default function NavItem( { item, postType, selected, statusLabel }: NavI
 				href={ editUrl }
 				target={ applyFilters( 'a8c.WpcomBlockEditorNavSidebar.linkTarget', undefined ) }
 			>
-				<div className="wpcom-block-editor-nav-item__title-container">
-					<div className={ titleClasses }>
-						{ item.title?.raw ||
-							_x( 'Untitled', 'post title for posts with no title', 'full-site-editing' ) }
-					</div>
-					{ item.slug && (
-						<div className="wpcom-block-editor-nav-item__slug">{ `/${ item.slug }/` }</div>
-					) }
+				<div className={ titleClasses }>
+					{ item.title?.raw ||
+						_x( 'Untitled', 'post title for posts with no title', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'Untitled', { context: 'post title for posts with no title'} )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/nav-sidebar-page-slug)